styles/globals.scss

```diff
@@ -9,6 +9,10 @@ body {
   margin: 0;
   font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Oxygen,
     Ubuntu, Cantarell, Fira Sans, Droid Sans, Helvetica Neue, sans-serif;
+
+    @media print {
+      font-size: 14px;
```

Comment:
I haven't used em or rem a ton before, but this was quite nice. Make a change at the root and all relative units update.